src/pages/Sketch/components/Toolbox/Toolbox.js

```diff
@@ -68,6 +69,9 @@ function Toolbox({
                 <Shape type_="text" id="sketch-shapes-text" label="Text">
                     <FaFont size={15}/>
                 </Shape>
+                <Shape type_="rightTriangle" id="sketch-shapes-righttriangle" label="Bi Shape Triangle">
```

Comment:
```suggestion
                <Shape type_="rightTriangle" id="sketch-shapes-rightTriangle" label="Bi Shape Triangle">
```
I know this is not at all necessary but it's better to update it everywhere it is been used. Just to make things look simpler.

Reply:
Yeah sure.

Reply:
Done.